mgmt/api/CoreAPI.h

```diff
@@ -45,10 +45,11 @@ TSProxyStateT ProxyStateGet();
 TSMgmtError ProxyStateSet(TSProxyStateT state, TSCacheClearT clear);
 TSMgmtError ServerBacktrace(unsigned options, char **trace);
 
-TSMgmtError Reconfigure();                            // TS reread config files
-TSMgmtError Restart(unsigned options);                // restart TM
-TSMgmtError Bounce(unsigned options);                 // restart traffic_server
-TSMgmtError StorageDeviceCmdOffline(const char *dev); // Storage device operation.
+TSMgmtError Reconfigure();                                                         // TS reread config files
+TSMgmtError Restart(unsigned options);                                             // restart TM
+TSMgmtError Bounce(unsigned options);                                              // restart traffic_server
+TSMgmtError StorageDeviceCmdOffline(const char *dev);                              // Storage device operation.
+TSMgmtError LifecycleMessage(char const *tag, void const *data, size_t data_size); // Lifecycle alert to plugins.
```

Comment:
Hahaha. Yes, I saw this as well and spent some time trying to fix my clang format before I realized I was reading the diff wrong. I couldn't figure out where this was coming from since the source file didn't have the split lines...